beamer/agent/agent.py

```diff
@@ -160,6 +161,7 @@ def _init(self) -> None:
         # This is necessary as we use the account for all resolutions and
         # would otherwise run into nonce problems
         self._task_pool = ThreadPoolExecutor(max_workers=1)
+        self._fill_mutexes: Tracker[tuple[ChainId, ChecksumAddress], threading.Lock] = Tracker()
```

Comment:
As mentioned before, we should not use `Tracker` here. Not only is it not necessary, but it is actively harmful because it has an implicit lock, which we do not want.